doc/fileexplorer.rst

```diff
@@ -2,22 +2,81 @@
 Files
 #####
 
-The **Files** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a filesystem and directory browser built right into Spyder.
+You can view and filter files according to their type and extension, open them with the :doc:`editor` or an external tool, and perform many common operations.
 
 .. image:: images/files/files-standard.png
-   :alt: Spyder File Explorer panel, showing a tree view of files and metadata
+   :alt: Spyder Files pane, showing a tree view of files and metadata
 
 
 
-==========================
-Performing file operations
-==========================
+===============
+File operations
+===============
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and opening an external file explorer.
+To browse the files on your system, use the arrows at the top of the pane.
+You can expand/collapse the folders in the pane to display the files and subdirectories hierarchically. 
+Double-clicking a folder will open it, showing the files inside and making it your working directory.
 
-.. image:: images/files/files-contextmenu-new.png
-   :align: center
-   :alt: File Explorer context menu, with common file manipulation tasks
+.. image:: images/files/files-browse.gif
+   :alt: Spyder Files pane showing browsing directories
+To open a file in the :guilabel:`Editor` from the :guilabel:`Files` pane, double-click its name.
```

Comment:
```suggestion

To open a file in the :guilabel:`Editor` from the :guilabel:`Files` pane, double-click its name.
```

Somehow both newlines got deleted here (as sometimes happens with Github suggestions, unfortunately), which is a syntax error in reST and is failing the build

Reply:
Still needs this one.